operator/parser/csv/csv_test.go

```diff
@@ -572,29 +572,176 @@ func TestParserCSV(t *testing.T) {
 	}
 }
 
-func TestParserCSVMultipleBodies(t *testing.T) {
-	t.Run("basic", func(t *testing.T) {
-		cfg := NewCSVParserConfig("test")
-		cfg.OutputIDs = []string{"fake"}
-		cfg.Header = testHeader
+func TestParserCSVMultiline(t *testing.T) {
+	cases := []struct {
+		name     string
+		input    string
+		expected map[string]interface{}
+	}{
+		{
+			"first_field",
+			"aa\naa,bbbb,cccc,dddd,eeee",
```

Comment:
Can we test with no newlines as well, and several lines? I would like to make sure existing parsing still works

Reply:
I added a basic case to this set.